### PR TITLE
MILESTONE 3: README AND LICENSE

### PR DESCRIPTION
**Pull request by JuniorGarciaRodriguez**
# In this PR I:
- Updated the license file
- Fixed broken link to license file
- Changed display name in **Authors** section of _README_ file
- Redesigned _README_ file for professional standards

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,8 +24,8 @@
 
 👤 **Junior Garcia**
 
-- GitHub: [@JuniorGarciaRodriguez](https://github.com/JuniorGarciaRodriguez)
-- Twitter: [@JGarciaGez](https://twitter.com/JGarciaGez)
+- GitHub: [@JuniorGarcia](https://github.com/JuniorGarciaRodriguez)
+- Twitter: [@JuniorGarcia](https://twitter.com/JGarciaGez)
```
Comment (Ibizugbe) [inline README.md:28]: Nice work setting up your readme👍 but you have omitted a lot of important sections, kindly use the [Microverse readme template](https://github.com/microverseinc/readme-template) and ensure all sections are added properly🤗🤗as it is a[ requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md) that our project is professionally documented🤗🤗

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @JuniorGarciaRodriguez 👋👋, 

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f69656d6d596a3332456945655a59766a70712f67697068792e676966


Good job so far!👍
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights✅✅

- [ ] Linters are passing👍
- [ ] Gihub flow practice has been used👍


## Required Changes ♻️

- [ ] Currently, your MIT license link is breaking and this can lead to a frustrating user experience and give our repo a less professional look😉😉 kindly fix this as it is a [requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md) that our project is professionally documented🤗
- [ ] Kindly add a more descriptive pull request title and heading as this will give your project a more professional look🤗 Currently your pull request description doesn't explain what has been implemented in this milestone😉 I suggest you go through [this lesson](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) as you implement the changes🤗🤗


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (JuniorGarciaRodriguez) [reply README.md:28]: Hello @Ibizugbe. I appreciate your time and advices. I still have one unclear doubt regarding the template. I think i missread it and deleted everything i didnt have a specific answer for as a "hello world" project. Do i have to come up with the information to cover all sections but FAQ (optional), or are there key sections i must always have in the projects?

> Modify this file to match your project and remove sections that don't apply

Comment (JuniorGarciaRodriguez) [reply README.md:82]: also cant find docs/vids on how to modify pull request message and microverse instructions says i cant start a new pull request but to use the same one. Could you help me with this as well?

**Review by brainconnect93 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @JuniorGarciaRodriguez, 👋 👋  

Your project is complete! Great job for following the project requirement🥇and implementing the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 




## Good Points 👍

- Has a proper Pull Request. ✔️
- No Linter errors ✔️
- Good use of Github flow ✔️

## Optional changes

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brainconnect93 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**